src/Admin/MessageAdmin.php

```diff
@@ -39,15 +39,15 @@ public function setMessageManager(MessageManager $messageManager)
     /**
      * @{inheritDoc}
      */
-    public function configureShowFields(ShowMapper $showMapper)
+    public function configureShowFields(ShowMapper $show): void
     {
         $showMapper->add('message');
```

Comment:
@7ochem Hier zit ook een foutje in: `$showMapper` vs. `$show`

Reply:
Ja idd, dat gaat niet goed